File: server/DTO/CommandeDTO.php
<?php


class CommandeDTO implements JsonSerializable
{
    private $idCommande;
    private $idTable;
    private $idProduit;
    private $nbProduit;

    public function __construct()
    {
    }

    public function getIdCommande()
    {
        return $this->idCommande;
    }

    public function setIdCommande($idCommande)
    {
        $this->idCommande = $idCommande;
    }

    public function getIdTable()
    {
        return $this->idTable;
    }

    public function setIdTable($idTable)
    {
        $this->idTable = $idTable;
    }

    public function getIdProduit()
    {
        return $this->idProduit;
    }

    public function setIdProduit($idPrduit)
    {
        $this->idProduit = $idPrduit;
    }


    public function getNbProduit()
    {
        return $this->nbProduit;
    }


    public function setNbProduit($nbProduit)
    {
        $this->nbProduit = $nbProduit;
    }


    public function jsonSerialize()
    {
        return array(
            'idCommande' => $this->idCommande,
            'idTable' => $this->idTable,
            'idProduit' => $this->idProduit,
            'nbProduit' => $this->nbProduit,
        );
    }

}

File: client/src/main/java/com/sio/pizzeria/DTO/CommandeDTO.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.sio.pizzeria.DTO;

/**
 *
 * @author Lenovo P50
 */
public class CommandeDTO {
    private int idCommande;
    private int idTable;
    private int idProduit;
    private int nbProduit;

    public CommandeDTO(int idTable, int idProduit, int nbProduit) {
        this.idTable = idTable;
        this.idProduit = idProduit;
        this.nbProduit = nbProduit;
    }
    
    
    
    public int getIdCommande() {
        return idCommande;
    }

    public void setIdCommande(int idCommande) {
        this.idCommande = idCommande;
    }

    public int getIdTable() {
        return idTable;
    }

    public void setIdTable(int idTable) {
        this.idTable = idTable;
    }

    public int getIdProduit() {
        return idProduit;
    }

    public void setIdProduit(int idProduit) {
        this.idProduit = idProduit;
    }

    public int getNbProduit() {
        return nbProduit;
    }

    public void setNbProduit(int nbProduit) {
        this.nbProduit = nbProduit;
    }
    
    
}
